packages/core/src/lib/config/normalizeConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { KeystoneConfig } from '../../types';
+
+/*
+  This function normalizes an input of KeystoneConfig or Promise<KeystoneConfig>
+  and just returns Promise<KeystoneConfig>
+*/
+
+export async function normalizeConfig(config: KeystoneConfig | Promise<KeystoneConfig>) : Promise<KeystoneConfig> {
+	return config;
```

Comment:
Alternatively, we can rewrite this with `Promise.resolve` instead of an `async function` to achieve the same thing.